modules/apps/product-navigation/product-navigation-personal-menu-web/src/main/resources/META-INF/resources/control_menu/personal_menu_icon.jsp

```diff
@@ -40,27 +41,9 @@
 
 <li class="control-menu-nav-item">
 	<span class="user-avatar-link">
-		<liferay-util:buffer
-			var="userAvatar"
-		>
-			<span class="sticker">
-				<span class="inline-item" id="personal-menu-icon-wrapper">
-					<liferay-ui:user-portrait
-						user="<%= user %>"
-					/>
-				</span>
-
-				<c:if test="<%= themeDisplay.isImpersonated() %>">
-					<span class="sticker sticker-bottom-right sticker-circle sticker-outside sticker-user-icon" id="impersonate-user-sticker">
-						<aui:icon id="impersonate-user-icon" image="user" markupView="lexicon" />
-					</span>
-				</c:if>
-			</span>
-		</liferay-util:buffer>
-
 		<liferay-product-navigation:personal-menu
 			expanded="<%= true %>"
-			label="<%= userAvatar %>"
+			user="<%= user %>"
```

Comment:
The element that triggers the personal menu needs to be customizable. If we're deprecating `label` here, we need an alternative way to set the custom trigger.

It's a feature requested here: https://issues.liferay.com/browse/LPS-90244

Reply:
Hey @pei-jung, is this a real feature request? The way it's used right now it doesn't look like this feature needs the whole label to be customizable, just to add support for the options you want.

Reply:
In any case, if this is a real feature request your team wants to offer and maintain, it should be as simple as allowing `label` to be passed down to the tag (as it is right now), and then using the value both in the `page.jsp` for the initial render as the value to `dangerouslySetInnerHTML` in the React component for the trigger.

Reply:
FYI, it's part of the requirements for https://issues.liferay.com/browse/LPS-90313